### ISSUE TITLE
No context menu on uploaded images/items

### ISSUE THREAD

**Issue by Reimeiki** [yannhodiesne/Discord-M1#34]
Default discord has this; is it feasible with this current m1 effort version? You can right click on other things which is great and things not directly uploaded into discord, but when its something like an image in discord, there is no right click for context menu ontop of an image to copylink/open in browser/save, etc. 
Thanks!

**Comment by tommasobenatti** [comment]
i can confirm

**Comment by jasmon-labs** [comment]
yes u won't get context menu but in order to get it meanwhile u have to click on the picture and then right click for context menu

**Comment by Reimeiki** [comment]
I suppose thats one way to go about it, but doesnt work for videos/music. But yeah, Eventually hopefully, its one of my most use features just being a context menu right click lol...
